### ISSUE TITLE
Automatically switch to slim view

### ISSUE THREAD

**Issue by dspacejs** [Taiko2k/TauonMusicBox#575]
Currently you can press F2 to toggle this view for songs:

![Screenshot from 2021-10-01 21-40-42](https://user-images.githubusercontent.com/9282837/135617672-114d2d3d-6f02-4af3-9720-e506d67fa112.png)

I think this should automatically be triggered once the window shrinks to a certain point. I can't see any reason why someone would want all the columns displaying when the window is tiny, because you can't see any of the content anyway. 

This would be similar to how responsive design behaves in web browsers.

**Comment by Taiko2k** [comment]
F2 switches the theme by default, I assume you mean toggling columns. Currently it does do that behavior when the gallery is open, but I suppose it makes sense for it to do this regardless of the view mode.

**Comment by dspacejs** [comment]
@Taiko2k yeah sorry I may have switched the key bindings a long time ago and forgot about it. I meant toggling columns.
